exercises/the-dice/canonical-data.json

```diff
@@ -0,0 +1,41 @@
+{
+  "exercise": "the-dice",
```

Comment:
The article "the" can be left out. Just `dice` would be better.

BUT, I think "dice" is not a good name for this problem, because it is not really about dice it is about detecting pairs.

There is an existing dice term two `1`s : [snake eyes](https://en.wikipedia.org/wiki/Snake_eyes)

I'm not aware of a term for two sixes, so I suggest changing the number that should be detected from `6` to `1` and calling the problem `snake-eyes` or `dice-snake-eyes`